// Copyright 2019 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// See platform2/media_perception/mojom/occupancy_trigger.mojom for relevant
// comments on data fields.

syntax = "proto2";

package mri;

message OccupancyTrigger {
  optional bool trigger = 1;

  optional int64 timestamp_ms = 2;
}
